CONFIGURATION.md

```diff
@@ -0,0 +1,256 @@
+# Wave Application Configuration
+
+Set Wave configuration values using environment variables or in [`config.yml`](./config.yml) configuration file
+
+### config.yml configuration
+
+Declare YAML configuration values in [`config.yml`](./config.yml)
+```
+wave:
+    mail:
+        from: "wave-app@seqera.io"
+```
+
+YAML configuration keys on this page are listed in "dot" notation, i.e., the mail from value in the snippet above is represented as `wave.mail.from` in the tables that follow.
+Environment variables for various attributes has been listed in third column, either you can provide those attributes values in config.yml or using corresponding environment variable.
+
+## General configuration
+
+General configuration options such as wave application name, port, whether to allow anonymous access (without tower token), wave and Seqera platform url.
+
+- The **`micronaut.application.name`** variable represents the name of the Wave application. By default, it is set to `wave-app`. *Optional*.
+
+- The **`micronaut.server.port`** variable determines the port used by the Wave server. The default port is `8080`. *Optional*.
+
+- The **`wave.allowAnonymous`** setting decides whether anonymous access to the Wave server is permitted. By default, it is set to `false`, meaning authentication is required. Change this option based on your security requirements. *Mandatory*.
```

Comment:
`wave.allowAnonymous` : says it has a default value, but also that is is mandatory. Sounds strange that the two both stand, can you doublecheck?